apply-labels/data/labeler-1/file.csv

```diff
@@ -0,0 +1,6 @@
+"Column-1","Column-2","Column-3","Column-n","Labeler","Question-label-1","Question-label-2","Question-label-3"
+"Data 1.1","Data 1.1","Data 3.1","Data 4.1","labeler@email.tld","Answer 1.1","Answer 2.1","Answer 3.1"
```

Comment:
keknya di backend blm ada answer validation ya?

aku coba submit answer yg ga valid (misal ga ada di option atau invalid date,time,url) tapi tetep ke submit, waktu re-submit lewat fe bakal kena form validationnya 
![Screenshot-20231214103127-275x287](https://github.com/datasaur-ai/api-client-example/assets/30997191/528f93c1-dc4c-4270-8f48-12a7b8f373d3)


<details>
<summary>Question set</summary>

```
[
  {
    "type": "TEXT",
    "config": {
      "multiple": false
    },
    "label": "Text",
    "required": false
  },
  {
    "type": "DROPDOWN",
    "config": {
      "multiple": false,
      "options": [
        {
          "id": "1",
          "label": "Dropdown 1",
          "parentId": null
        },
        {
          "id": "2",
          "label": "Dropdown 2",
          "parentId": null
        },
        {
          "id": "3",
          "label": "Dropdown 2.1",
          "parentId": "2"
        },
        {
          "id": "4",
          "label": "Dropdown 2.1.1",
          "parentId": "3"
        }
      ]
    },
    "label": "Dropdown",
    "required": false
  },
  {
    "type": "DATE",
    "config": {
      "format": "DD/MM/YYYY",
      "defaultValue": null,
      "multiple": false
    },
    "label": "Date",
    "required": false
  },
  {
    "type": "TIME",
    "config": {
      "format": "HH:mm",
      "defaultValue": null,
      "multiple": false
    },
    "label": "Time",
    "required": false
  },
  {
    "type": "CHECKBOX",
    "config": {
      "hint": "",
      "multiple": false
    },
    "label": "Checkbox",
    "required": false
  },
  {
    "type": "RADIO",
    "config": {
      "hint": "",
      "options": [
        {
          "id": "1",
          "label": "True"
        },
        {
          "id": "2",
          "label": "False"
        }
      ]
    },
    "label": "Radio",
    "required": false
  },
  {
    "type": "SLIDER",
    "config": {
      "min": 1,
      "max": 10,
      "step": 1,
      "gradientColors": [
        "#538CFF",
        "#538CFF"
      ],
      "hideScaleLabel": false,
      "multiple": false
    },
    "label": "Slider",
    "required": false
  },
  {
    "type": "URL",
    "config": {
      "multiple": false
    },
    "label": "URL",
    "required": false
  }
]
```

</details>

<details>
<summary>Answer</summary>

```
"Column-1","Column-2","Column-3","Column-n","Labeler","Text","Dropdown","Date","Time","Checkbox","Radio","Slider","URL"
"Data 1.1","Data 1.1","Data 3.1","Data 4.1","labeler@email.tld","Answer 1.1.1","Dropdown.1","2023-11-11-11","11:11:11:11","11","11","11","/1"
```

</details>

Reply:
resolved ya, will be handled separately 👍🏼